_includes/macros.html

```diff
@@ -285,3 +290,62 @@ <h1 class="h5 {{'u-visually-hidden' if hide_heading}}">Share this page</h1>
 
 {% endmacro %}
 
+{# ==========================================================================
+
+   youtube()
+
+   ==========================================================================
+
+   Description:
+
+   Builds Youtube markup when given:
```

Comment:
See here and other occurrences in the img alt text and JS. It should probably be consistent with https://github.com/cfpb/cfgov-refresh/pull/528/files#diff-1bd0e4b03b792eb67fdeed318c7d8385R60